app/routes.js

```diff
@@ -0,0 +1,25 @@
+var	express = require('express');
+var	config = require('./controller/settings');
+
+var	router = express.Router();
+
+// Route to UPDATE the config
+router.post('/configs', function (req, res) {
+	
+	var configData = config.update(req);
+	console.log(configData);
+	res.send(config.update(req));
```

Comment:
I'm now using configData.
this should be a pram object of update not a new var - > This is not clear to me, can you describe it a little.